frontend/src/app/components/search/search.component.html

```diff
@@ -47,7 +47,7 @@
     <div class="search-grid flex-row">
       <md-icon>search</md-icon>
       <md-input-container floatPlaceholder="never" class="search-box">
-        <input #searchBox mdInput placeholder="Name, Username or Roll" (keyup)="searchTerm(searchBox.value)" [disabled]="loading">
+        <input #searchBox mdInput placeholder="Name, Username or Roll" type="text" (keyup)="searchTerm(searchBox.value)">
```

Comment:
Why have we removed the `[disabled]` ?? This allows people to type before the page is loaded. (Bad UX)